charts/infra-apps/values.yaml

```diff
@@ -111,7 +111,7 @@ ingressNginx:
   # -- Chart
   chart: ingress-nginx
   # -- [ingress-nginx Helm chart](https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx) version
-  targetRevision: 4.2.*
+  targetRevision: 4.4.0
```

Comment:
why `4.4.0`  and not `4.4.*`  ?

Reply:
because i've been slowly getting rid of wildcards with each update i touch, not sure when we started but this isn't the first wildcard we're getting rid of by a long stretch and no more wildcards makes our stuff  much more predictive wrt to ArgoOutOfSync only happening when it makes sense.